### ISSUE TITLE
PR/edit button

### ISSUE THREAD

**Issue by AdriVanHoudt** [hapijs/hapijs.com#497]
I think it would boost PRs/edits if people would be encouraged to edit the page they are looking at

**Comment by Marsup** [comment]
Not sure what you're suggesting, isn't that already a feature of github ?

**Comment by devinivy** [comment]
I think he means on the site itself it should link to the file in github that you can edit to suggest changes.

**Comment by Marsup** [comment]
Oh, I thought I was on contrib, oops :p

**Comment by AdriVanHoudt** [comment]
yeah on the website itself

**Comment by Nargonath** [comment]
I'll have a try at this issue.

**Comment by Nargonath** [comment]
I thought on putting the button in header right on the navigation bar:
![image](https://user-images.githubusercontent.com/583657/46919949-67c29300-cfe7-11e8-9f9a-0f9871100c7c.png)

What do you think? Of course it would be contextualize depending on the page you are viewing. 😃

**Comment by devinivy** [comment]
I personally think that looks nice.

**Comment by Nargonath** [comment]
Concerning the content that would be linked to for most of the pages it was their associated pug template. In some cases it did not make sense so I took the liberty to link to the most related content:
* `/tutorials` are linked to the proper `.md` file according to the language and the section selected.
* `/api` is linked to the `API.md` on hapi repository either on the `master` branch for v17 or `v16` branch for latest `v16`.
* `/community`, `/plugins` and `/resources` are linked to their respective `lib/.js` file because all the contents is there.

I also linked the pages that are available even though not directly linked from the website: `/governance`, `/styleguide`, `/operating` and `/hapidays`. The latter is pretty out of date but I figured I'd link it anyway.
